project/nef.xcodeproj/xcshareddata/xcschemes/Markdown.xcscheme

```diff
@@ -50,6 +50,16 @@
             ReferencedContainer = "container:nef.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "--project ~/Desktop/Documentation.app"
+            isEnabled = "YES">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "--output ~/Desktop/Migue2"
+            isEnabled = "YES">
+         </CommandLineArgument>
+      </CommandLineArguments>
```

Comment:
This information should not be committed to the repository, as it is local to your computer.